docs/data-ai/inference/augment-visual-inference-with-sensor-data.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
The Run inference section is now very long compared to other sections. Consider grouping them (Maybe Detecting & Tracking, Augmenting, Autonomous decisions?)